CraftTweaker2-API/src/main/java/crafttweaker/api/event/EntityLivingEquipmentChangeEvent.java

```diff
@@ -0,0 +1,26 @@
+package crafttweaker.api.event;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.entity.IEntityEquipmentSlot;
+import crafttweaker.api.item.IItemStack;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenGetter;
+import stanhebben.zenscript.annotations.ZenMethod;
+import stanhebben.zenscript.annotations.ZenSetter;
+
+@ZenClass("crafttweaker.event.EntityLivingEquipmentChangeEvent")
+@ZenRegister
+public interface EntityLivingEquipmentChangeEvent extends ILivingEvent {
+
+	@ZenMethod
+    @ZenGetter("olditem")
+    IItemStack getFrom();
+
+    @ZenMethod
+    @ZenGetter("item")

```

Comment:
Maybe this should be `newItem`? since you called the old item `oldItem` just so it is clearer to users